alfresco-transform-imagemagick/alfresco-transform-imagemagick-boot/src/main/java/org/alfresco/transformer/ImageMagickController.java

```diff
@@ -79,7 +81,16 @@
 {
     private static final Logger logger = LoggerFactory.getLogger(ImageMagickController.class);
 
-    private ImageMagickCommandExecutor commandExecutor = new ImageMagickCommandExecutor();
+    @Autowired
+    private GlobalProperties externalProps;
```

Comment:
Could we just grab the values like @Value("the.value.property : default.value.property")